src/main.js

```diff
@@ -533,19 +560,14 @@ async function ensureMicrophoneRunning() {
     return true;
 }
 
-async function setupMicrophone(channel) {
+async function setupMicrophone() {
     const $micPermissionStatus = $("#micPermissionStatus");
     $micPermissionStatus.text("Requesting microphone access...");
 
     // Try to initialise the microphone
     const success = await microphoneInput.initialise();
     if (success) {
-        console.log("Microphone: Successfully initialised from URL parameters");
-        // Set microphone as source for its channel
-        console.log(`Setting microphone as source for channel ${channel}`);
-        processor.adconverter.setChannelSource(channel, microphoneInput);
         $micPermissionStatus.text("Microphone connected successfully");
```

Comment:
We appear to have lost this functinoality? Can you explain how the `$micPermissionStatus` is updated, or is it now defunct?